src/main/java/seedu/address/ui/PositionListPanel.java

```diff
@@ -35,53 +27,7 @@
     public PositionListPanel(ObservableList<Position> positionList) {
         super(FXML);
 
-        Set<Requirement> reqs1 = new HashSet<>();
-        reqs1.add(new Requirement("Java"));
-        reqs1.add(new Requirement("C++"));
-
-        Position tempPos1 = new Position(
-                new PositionName("Senior Software Developer"),
-                new Description("The highest paying job in the company\n"
-                        + "More than 5 years experience"),
-                new PositionOpenings("3"),
-                reqs1);
-
-        Set<Requirement> reqs2 = new HashSet<>();
-        reqs2.add(new Requirement("Source Academy"));
-        reqs2.add(new Requirement("C"));
-
-        Position tempPos2 = new Position(
-                new PositionName("Useless IT Intern"),
-                new Description("Need to hire to fill the quota"),
-                new PositionOpenings("1"),
-                reqs2);
-
-        Set<Requirement> reqs3 = new HashSet<>();
-        reqs3.add(new Requirement("Sweep Floor"));
-        reqs3.add(new Requirement("Wipe Window"));
-        reqs3.add(new Requirement("Wash Toilet"));
-
-        Position tempPos3 = new Position(
-                new PositionName("Janitor"),
-                new Description("Arguably the most important job"),
-                new PositionOpenings("0"),
-                reqs3);
-
-        Position tempPos4 = new Position(
-                new PositionName("Admin Officer"),
-                new Description("Degree or Postgraduate holder with Major in Information Technology, "
-                        + "Computer Science, or other similar focus, and a cumulative GPA of 3.5 and above"),
-                new PositionOpenings("1"),
-                new HashSet<>());
-
-        List<Position> tempList = new ArrayList<>();
-        tempList.add(tempPos1);
-        tempList.add(tempPos2);
-        tempList.add(tempPos3);
-        tempList.add(tempPos4);
-        ObservableList<Position> posList = FXCollections.observableList(tempList);
-
-        positionListView.setItems(posList); // TO REPLACE WITH REAL LIST
+        positionListView.setItems(positionList); // TO REPLACE WITH REAL LIST
```

Comment:
Similar to above. Would removing the comment make it clearer for the rest of the developers?